packages-next/fields/src/types/decimal/index.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
why add it at the top, you can add it where it is needed.

Reply:
It's not to the whole file(`@ts-ignore` doesn't work like that), it's to the import on the next line.

Reply:
I simply copy-pasted this trick from `packages-next/fields/src/types/integer/index.ts` file :)